servlet/explainingHttpServlet/src/main/java/jakartaee/examples/servlet/explainingHttpServlet/ExplainingHttpServlet.java

```diff
@@ -10,14 +10,14 @@
  * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
  * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
-package jakartaee.examples.servlet.thelearningservlet;
+package jakartaee.examples.servlet.explainingHttpServlet;
 
 /**
- * TheLearningServlet Web Application
+ * ExplainingHttpServlet Web Application
  *
- * Prepared by Ken Fogel omniprof@gmail.com
+ * @author Ken Fogel (omniprof@gmail.com)
  *
- * Twitter: @omniprof - Blog: www.omnijava.com
+ * Twitter: omniprof - Blog: www.omnijava.com
```

Comment:
Sorry I missed this one ... none of the examples add more than the @author tag so can you remove this and then I'll approve the PR and I'll ask @arjantijms Arjan to merge it.